app/pages/patients/patients.js

```diff
@@ -53,7 +49,7 @@ export let Patients = React.createClass({
   render: function() {
     var welcomeTitle = this.renderWelcomeTitle();
 
-    if (this.isLoading() && !(this.props.showingWelcomeTitle && this.props.showingWelcomeSetup)) {
+    if (this.isLoading() && !(this.props.showingWelcomeMessage)) {
```

Comment:
I had added this second condition and now I think we want to remove it... We _do_ want the "loading" message to render while waiting if we're showing the welcome setup.